File: crop-me/src/main/java/com/mikelau/croperino/AlertInterface.java
package com.mikelau.croperino;

import android.content.DialogInterface;

public class AlertInterface {

    public interface WithNeutral {
        void PositiveMethod(DialogInterface dialog, int id);
        void NeutralMethod(DialogInterface dialog, int id);
        void NegativeMethod(DialogInterface dialog, int id);
    }
}

File: croperino/src/main/java/com/mikelau/croperino/AlertInterface.kt
package com.mikelau.croperino

import android.content.DialogInterface

interface WithNeutral {
    fun PositiveMethod(dialog: DialogInterface?, id: Int)
    fun NeutralMethod(dialog: DialogInterface?, id: Int)
    fun NegativeMethod(dialog: DialogInterface?, id: Int)
}

